x-pack/plugins/infra/server/lib/adapters/metrics/kibana_metrics_adapter.ts

```diff
@@ -83,7 +83,7 @@ export class KibanaMetricsAdapter implements InfraMetricsAdapter {
               series: panel.series.map((series) => {
                 return {
                   id: series.id,
-                  label: series.label,
+                  label: [series.label].flat()[0],
```

Comment:
Is this change intentional? 
As far as I can understand this will have no effect?

Reply:
Yes, this is by design to avoid type errors on your end. When using splitting by one field, everything will continue to work as before

Reply:
Makes sense, if you're making other changes it would be great if you could include that context in a code comment!